lib/src/analyzers/lint_analyzer/models/issue.dart

```diff
@@ -43,4 +43,14 @@ class Issue {
     this.verboseMessage,
     this.suggestion,
   });
+
+  @override
+  String toString() => 'Issue{'
```

Comment:
Could you point out how this change is used?

Reply:
Logging for debugging what is that issue. I can revert if needed.

Reply:
reverted